charts/db-operator/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 type: application
 ## All supported k8s versions are in the test: https://github.com/db-operator/db-operator/blob/master/.github/workflows/build-and-test.yaml
-kubeVersion: ">= 1.21-prerelease"
```

Comment:
this is a bit misleading, but the `-prerelease` text is needed, or you unable to install this in gke, related google has an version suffix, misunderstand by helm and think it's incompatible.